### PR TITLE
Abhishek pascal string

### PR DESCRIPTION
**Pull request by Abhishek08Mi**

### CODE REVIEW

File: abhishek/pascal_string/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+P_string_output: p_string.o Driver.o
+	gcc p_string.o Driver.o -o P_string_output
+p_string.o: p_string.c 
+	gcc -c p_string.c
+Driver.o: Driver.c 
+	gcc -c Driver.c 
+clean:
+	rm -rf *.o P_string_output
+	rm -rf *.exe P_string_output
```
Comment (aloknigam247) [inline abhishek/pascal_string/Makefile:9]: what do we need to clear .exe here ?

File: abhishek/pascal_string/Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+P_string_output: p_string.o Driver.o
+	gcc p_string.o Driver.o -o P_string_output
+p_string.o: p_string.c 
+	gcc -c p_string.c
+Driver.o: Driver.c
```
Comment (aloknigam247) [inline abhishek/pascal_string/Makefile:5]: use main.c as driver

File: abhishek/pascal_string/Makefile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+P_string_output: p_string.o Driver.o
```
Comment (aloknigam247) [inline abhishek/pascal_string/Makefile:1]: program name should be demo

**Review by aloknigam247 — CHANGES_REQUESTED** [review]

File: abhishek/pascal_string/Makefile (hunk 3 above)
Comment (Abhishek08Mi) [reply abhishek/pascal_string/Makefile:1]: Sir Program name means target?

File: abhishek/pascal_string/Makefile (hunk 2 above)
Comment (Abhishek08Mi) [reply abhishek/pascal_string/Makefile:5]: Sir instead of driver.o i should take main.o?

File: abhishek/pascal_string/Makefile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+demo: p_string.o main.o
+	gcc p_string.o main.o -o demo
+p_string.o: p_string.c 
+	gcc -c p_string.c
+main.o: main.c 
+	gcc -c Driver.c
```
Comment (aloknigam247) [inline abhishek/pascal_string/Makefile:6]: Driver.c ?

**Review by aloknigam247 — CHANGES_REQUESTED** [review]

**Pull request merged**